### DISCUSSION TITLE
Transcribing Telegram voice message using Whisper

### DISCUSSION THREAD

**Discussion by ckaytev** [openai/whisper#933] [Show and tell]
Seeing a [message](https://github.com/openai/whisper/discussions/294) from @bjnortier, I decided to write a telegram bot to recognize voice messages.

I don't like receiving audio messages in chat so these are great solutions for me. I hope you appreciate
https://github.com/ckaytev/tgisper

The plans are to build the code into a container with automatic deployment in the cloud.

I'm waiting for your feedback

**Comment by arvbs** [comment]
Telegram has a built-in function in the premium account. 

https://telegram.org/faq_premium

**Reply by Lemon553311-dev** [reply]
well duh that one is free

**Reply by gadimich** [reply]
there is also an Audio to Audio translation bot - https://tryrespeak.com
